README.md

```diff
@@ -541,3 +542,31 @@ There is a WIP GUI which can be started via (from the root folder) `cargo r --bi
 
 It's quite barebones right now, but the goal is the be able to attach to any running Serial Keel instance and see the
 user events and also the raw logs from each endpoint in real time.
+
+## Serial Keel as a systemd service
+
+You can setup Serial Keel to run as a systemd service on startup. In `scripts/systemd`, you'll find a template `.service`
+file and a setup script.
+
+The script assumes that you have a valid serial-keel configuration file in your user home directory named `config.ron`,
```

Comment:
It would be great if the script would use `~/config.ron` as the default but allow passing it in as well.

Reply:
Done :+1:

Reply:
Nice! Docs should be updated to match :slightly_smiling_face: